queries.sql

```diff
@@ -115,4 +115,24 @@ JOIN vets vt ON v.vets_id = vt.id
 WHERE vt.name = 'Maisy Smith'
 GROUP BY s.name, vt.name
 ORDER BY num_visits DESC
-LIMIT 1; 
\ No newline at end of file
+LIMIT 1; 
+
+-- Week 2
+SELECT COUNT(*) FROM visits where animals_id = 4;
+SELECT * FROM visits where vets_id = 2;
+SELECT * FROM owners where email = 'owner_18327@mail.com';
+
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits where animals_id = 4;
+EXPLAIN ANALYZE SELECT * FROM visits WHERE vets_id = 2;
+EXPLAIN ANALYZE SELECT * FROM owners WHERE email = 'owner_18327@mail.com';
+
+CREATE INDEX idx_animal_id ON visits (animals_id);
+ANALYZE visits;
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits where animals_id = 4;
+
+CREATE INDEX idx_vets_id ON visits (vets_id);
+ANALYZE visits;
+EXPLAIN ANALYZE SELECT id,animals_id,date_of_visit FROM visits where vets_id = 2;
+CREATE INDEX idx_email ON owners (email);
+ANALYZE visits;
+EXPLAIN ANALYZE SELECT id,full_name FROM owners where email = 'owner_18327@mail.com';
```

Comment:
-  You have done great so far. :+1: :+1: But there are a few changes you need to make before moving to the next milestone. :smile: :small_airplane: 
- [ ] You did great using `indexes` to speed up queries in the database, but following the best practices, the creation of `TABLES, COLUMNS OR INDEXES` should be done in the `schema.sql` file. I'll suggest you move the index creation SQL lines to the correct file. :+1: :100: